spec/lib/stats_reporter_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'stats_reporter'
+require File.expand_path(File.dirname(__FILE__) + '../../../config/boot')
```

Comment:
boot should be loaded by `spec_helper`, which is required universally by `.rspec`, so you shouldn't need to require it again.